StortSpelProjekt/Engine/src/ECS/Components/Lights/PointLightComponent.cpp

```diff
@@ -69,8 +69,9 @@ namespace component
 		return m_pPointLight;
 	}
 
-	void PointLightComponent::UpdateLightColor()
+	void PointLightComponent::UpdateLightColorIntensity()
 	{
 		m_pPointLight->baseLight.color = m_pBaseLight->color;
+		m_pPointLight->baseLight.intensity = m_pBaseLight->intensity;
 	}
```

Comment:
Do not see the reason why this is not in Light class, it has the same code in all lights.

Reply:
Also not sure why it is not called Update(), or just UpdateLight.

Reply:
Not sure what the point of this function is either, it is called on SetIntensity(), then you set the color which is unchanged in this function. Also don't quite understand the baseLight vs m_pBaseLight.

Reply:
It is in the baselight